test/typescript/index.ts

```diff
@@ -1 +1,14 @@
 import * as rollup from './dist/rollup';
+
+// Plugin API
+interface Options {
+	extensions?: string | string[];
+}
+const plugin: rollup.PluginImpl<Options> = (options = {}) => {
+	// tslint:disable-next-line:no-unused-variable
+	const extensions = options.extensions || ['.js'];
+	return { name: 'my-plugin' };
+};
+
+plugin();
+plugin({ extensions: ['.js', 'json'] });
```

Comment:
Nice idea adding a test that demonstrates the basic usage! Maybe this should be added to the wiki as a recommendation for plugins written in TypeScript https://github.com/rollup/rollup/wiki/Plugins#conventions once it is released?

Reply:
I'll do that, great suggestion, thx !